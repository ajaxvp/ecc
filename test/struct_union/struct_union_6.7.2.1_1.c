/* ISO: 6.7.2.1; constraint tests */

struct b
{
    // ISO: 6.7.2.1 (3)
    int x : 5.0f;
    
    // ISO: 6.7.2.1 (3)
    int y : -3;

    // ISO: 6.7.2.1 (3)
    int z : 33;

    // ISO: 6.7.2.1 (3)
    int a : 0;

    // ISO: 6.7.2.1 (4)
    float f : 1;
};
