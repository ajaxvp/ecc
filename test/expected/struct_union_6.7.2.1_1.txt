ecc: error: [6:13] struct and union bitfields are not supported yet
ecc: error: [6:13] bitfield width must be an integer constant expression
ecc: error: [9:13] struct and union bitfields are not supported yet
ecc: error: [9:13] bitfield width must be nonnegative
ecc: error: [12:13] struct and union bitfields are not supported yet
ecc: error: [12:13] bitfield width must not exceed the typical width of its declaring type
ecc: error: [15:13] struct and union bitfields are not supported yet
ecc: error: [15:9] zero-width bitfields may not declare an identifier
ecc: error: [18:15] struct and union bitfields are not supported yet
ecc: error: [18:11] bitfield must have a type of bool, int, or unsigned int
